### PR TITLE
[FLINK-18253][doc][avro] Add filesystem option documentation for Avro

### PR DESCRIPTION
**Pull request by JingsongLi**
## What is the purpose of the change

Add filesystem option documentation for Avro

## Verifying this change

image

### CODE REVIEW

File: docs/dev/table/connectors/formats/avro.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ Format Options
       <td>String</td>
       <td>Specify what format to use, here should be 'avro'.</td>
     </tr>
+    <tr>
+      <td><h5>avro.codec</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>For filesystem connector only, the compression codec for avro. Support deflate, snappy, bzip2, xz.</td>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/avro.md:105]: Add a link to 'filesystem connector' and describe the default behavior.

File: docs/dev/table/connectors/formats/avro.md [hunk 2]
```diff
@@ -97,6 +97,13 @@ Format Options
       <td>String</td>
       <td>Specify what format to use, here should be 'avro'.</td>
     </tr>
+    <tr>
+      <td><h5>avro.codec</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>For <a href="{{ site.baseurl }}/dev/table/connectors/filesystem.html">Filesystem</a> only, the compression codec for avro. Default no compression. Support deflate, snappy, bzip2, xz.</td>
+    </tr>
```
Comment (danny0405) [inline docs/dev/table/connectors/formats/avro.md:106]: No compression as default. The valid enumerations are: `deflate`, `snappy`, `bzip2`, `xz`.

Comment (wuchong) [reply docs/dev/table/connectors/formats/avro.md:106]: Let's use `{% link dev/table/connectors/filesystem.md %}` as the link which is suggested in the dev mailing list recentlly.

**Comment by JingsongLi** [comment]
Thanks @wuchong @danny0405 for review, updated.

**Pull request merged**